mypyc/test-data/run-dicts.test

```diff
@@ -192,3 +193,18 @@ else:
 2
 4
 2
+
+[case testDictMethods]
+from collections import defaultdict
+from typing import Dict
+
+def test_dict_copy() -> None:
+    d: Dict[str, int] = {}
+    assert d.copy() == d # type: ignore
+    d = {'a': 1, 'b': 2}
+    assert d.copy() == d # type: ignore
+    assert d.copy() is not d # type: ignore
+    dd: Dict[str, int] = defaultdict(int)
+    dd['a'] = 1
+    assert dd.copy() == dd # type: ignore
```

Comment:
It would be good to get rid of these `# type: ignore` comments as well (see above).

Reply:
Removed.